src/Bluejay.asm

```diff
@@ -1022,6 +1042,15 @@ ENDIF
     ljmp motor_start                    ; Go back and try starting motors again
 
 exit_run_mode_stall_done:
+    ; Clear extended DSHOT telemetry flag if turtle mode is not active
+    ; This flag is also used for EDT safety arm flag
+    ; We don't want to deactivate extended telemetry during turtle mode
+    ; Extended telemetry flag is important because it is involved in
+    ; EDT safety feature. We don't want to disable EDT arming during
+    ; turtle mode.
+    jb Flag_User_Reverse_Requested, ($+5)
```

Comment:
can we have a jump to a label here instead of a relative jump?

Reply:
https://github.com/bird-sanctuary/bluejay/issues/109